interface-definitions/include/qos/class-match.xml.i

```diff
@@ -69,9 +69,6 @@
               <format>802.1Q</format>
               <description>802.1Q VLAN tag</description>
             </valueHelp>
-            <constraint>
```

Comment:
Removing a constraint is IMHO not a wise idea. As `constraint` operates in logical OR mode, this could be fixed by:

```xml
            <constraint>
              <validator name="numeric" argument="--range 1-65535"/>
              <validator name="ip-protocol"/>
            </constraint>
```

Unfortunately suggestions are not supported on deleted lines.

Reply:
`<validator name="ip-protocol"/>` don't have all available protocols for ether match, I create a new similar validator but for ether types. The list of allowed values gets from [vyatta-cfg-qos](https://github.com/vyos/vyatta-cfg-qos/blob/current/templates-skeleton/qos-match-templates/class/node.tag/match/node.tag/ether/protocol/node.def)